deploy/docker/start.sh

```diff
@@ -1,47 +1,41 @@
 #!/bin/sh
-set -e
+set -e -u
 
 emqx_exit(){
-    # tail erlang.log.*
-    erlang_log=$(echo $(ls -t /opt/emqx/log/erlang.log.*) | awk '{print $1}')
-    num=$(sed -n -e '/LOGGING STARTED/=' ${erlang_log} | tail -1)
-    tail -n +$((num-2)) ${erlang_log}
+    # tail emqx.log.*
+    num=$(sed -n -e '/LOGGING STARTED/=' /opt/emqx/log/emqx.log | tail -1)
+    tail -n +$((num-2)) /opt/emqx/log/emqx.log
```

Comment:
use `$(expr num - 2)` here too ?
I wonder what happens when `num` is less than 3

Reply:
Do you think we still need the `emqx_exit` method?
Because we have executed `tail -f /opt/emqx/log/emqx.log &` on line 27
Do we just execute `/opt/emqx/bin/emqx start || tail -f /opt/emqx/log/erlang.log.1` on line 19 ?

Reply:
Now I am not entirely sure if erlang.log and emqx.log is the same thing.
is erlang.log where `runerl` pipes to, and `emqx.log` is where the logging backend writes to ?

Reply:
Use `tail -f /opt/emqx/log/erlang.log.1` on line 19 because if `emqx start` fails, the `emqx.log` file does not necessarily exist

Reply:
yes exactly, when emqx fails to start, it's interesting to see what's in `erlang.log` and `emqx.log` may not exist 
I guess this is what this `emqx_exit` function invented for.
I.e. We should not change it to print `emqx.log`

Reply:
The `emqx_exit` function is to find the correct file content and print it when there are multiple `erlang.log.*`, when `emqx start` fails, there should be only `erlang.log.1`, maybe `emqx_exit` function is no need at the end of the file

Reply:
yes, but you changed to print emqx.log, see line 6:
`num=$(sed -n -e '/LOGGING STARTED/=' /opt/emqx/log/emqx.log | tail -1)`

Reply:
with regards to `erlang.log.X` vs `erlang.log.1`, I guess this function was copied from non-container start script.
I have no strong opinion wrt whether or not we should change to `.1` only

Reply:
`num=$(sed -n -e '/LOGGING STARTED/=' /opt/emqx/log/emqx.log | tail -1)` It is a wrong way of writing, because only "LOGGING STARTED" is included in erlang.log, This is to find the log of the last startup when emqx is started multiple times

If delete the `emqx_exit` function, then don't need to keep this line anymore

Reply:
I understand what the greping of `LOG STARTED` does here.
emqx_exit was a function because it was called in two places (see the last line),
this perhaps also imply that it's not always `.1`